lib/webdriveragent.js

```diff
@@ -182,7 +182,9 @@ class WebDriverAgent {
         return;
       }
       if (isPertinentLogLine(out)) {
-        agentLog.debug(`${prefix}: ${out.trim()}`);
+        for (let line of out.split("\n")) {
+          agentLog.debug(`${prefix}: ${line.trim()}`);
```

Comment:
Splitting on `\n` gets rid of the newline, so I don't think we ought to do the `trim()` here. With it everything will be left-justified, and merge together. Without it the indentation that the iOS log produces will be maintained.

```
dbug WebDriverAgent Sim: Aug 18 17:53:59 isaac CoreSimulatorBridge[35498]: Requesting installation of file:///Users/isaac/Library/Developer/Xcode/DerivedData/WebDriverAgent-eoyoecqmiqfeodgstkwbxkfyagll/Build/Products/Debug-iphonesimulator/WebDriverAgentRunner-Runner.app/ with options: {
dbug WebDriverAgent Sim: CFBundleIdentifier = "com.apple.test.WebDriverAgentRunner-Runner";
dbug WebDriverAgent Sim: PackageType = Developer;
dbug WebDriverAgent Sim: SimulatorRootPath = "/Applications/Xcode-7.3.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk";
dbug WebDriverAgent Sim: SimulatorUserPath = "/Users/isaac/Library/Developer/CoreSimulator/Devices/F07AB371-A73B-4C3F-9A85-CD48F9986B61/data";
dbug WebDriverAgent Sim: }
```
vs
```
dbug WebDriverAgent Sim: Aug 18 17:54:57 isaac CoreSimulatorBridge[35798]: Requesting installation of file:///Users/isaac/Library/Developer/Xcode/DerivedData/WebDriverAgent-eoyoecqmiqfeodgstkwbxkfyagll/Build/Products/Debug-iphonesimulator/WebDriverAgentRunner-Runner.app/ with options: {
dbug WebDriverAgent Sim: 	    CFBundleIdentifier = "com.apple.test.WebDriverAgentRunner-Runner";
dbug WebDriverAgent Sim: 	    PackageType = Developer;
dbug WebDriverAgent Sim: 	    SimulatorRootPath = "/Applications/Xcode-7.3.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk";
dbug WebDriverAgent Sim: 	    SimulatorUserPath = "/Users/isaac/Library/Developer/CoreSimulator/Devices/F07AB371-A73B-4C3F-9A85-CD48F9986B61/data";
dbug WebDriverAgent Sim: 	}
```